python/tk_desktop/desktop_window.py

```diff
@@ -799,8 +800,9 @@ def __launch_app_proxy_for_project(self, project, pipeline_configuration_id=None
                 "proxy_auth": server_auth,
             },
         }
-        (_, pickle_data_file) = tempfile.mkstemp(suffix='.pkl')
+        (fd, pickle_data_file) = tempfile.mkstemp(suffix='.pkl')
         pickle.dump(desktop_data, open(pickle_data_file, "wb"))
```

Comment:
I didn't modify that section so I wasn't looking to change anything else but the file handle issue. If you want me to also change that to add a `try/except` I will. Just trying to keep this focused on one issue.